addons/project-folders/userstyle.css

```diff
@@ -0,0 +1,51 @@
+.folders {
+  padding: 10px;
+}
+
+.folder {
+  width: 100px !important;
+  border: 1px black solid !important;
+  text-align: center !important;
+  border-radius: 5px !important;
+  height: 90px !important;
+  padding: 5px !important;
+  display: inline-block !important;
+  margin: 10px !important;
+  background-color: #fcfcfc !important;
+  color: black !important;
```

Comment:
Why is the extra specificity required? Aren't you creating this element yourself?

Reply:
I did that because I also added a `.button` class `.folder`. But I will check if `!important` is still required or not

Reply:
It is fixed in the latest commit